docker-rke2.yaml

```diff
@@ -156,7 +158,7 @@ data:
       http-check expect status 401
```

Comment:
Also we can probably add fixes for LB healthcheck to use `200` code here, which will check for `ETCD` health as well - https://github.com/rancher/cluster-api-provider-rke2/commit/bb2410b647c8189520df78efe0e8c7a72f6394cf